### DISCUSSION TITLE
2.0 first beta release

### DISCUSSION THREAD

**Discussion by tonioo** [modoboa/modoboa#2348] [Upgrade]
Brings a new interface built with vue.js and and a new API version.

<hr /><em>This discussion was created from the release <a href='https://github.com/modoboa/modoboa/releases/tag/2.0.0-beta.1'>2.0 first beta release</a>.</em>

**Comment by AmazingBeu** [comment]
I'm trying to upgrade my modoboa to the beta on debian 10.

In case someone encounters the problem, I had to upgrade pip to version 21.2.4 to be able to upgrade the modoboa version, 

Then, I have a problem, I don't have a folder /srv/modoboa/env/lib/python3.7/site-packages/frontend, any idea @tonioo ?

**Reply by tonioo** [reply]
Yes, I've discovered that. This is a packaging issue, I'm trying to fix it.

**Reply by tonioo** [reply]
I think I've found a solution: https://github.com/modoboa/modoboa/commit/f234dc9544684f32d57891eb024735684f5a5923
But it will require a new beta release.

**Reply by AmazingBeu** [reply]
Ok, I will wait for the version :grin:

**Reply by tonioo** [reply]
A beta2 release is available and it looks better ;)

**Comment by duderuud** [comment]
Installed the beta on a clean Ubuntu 20.04 VM but can't login. The login screen just keeps loading.
Can't find any logging related to it, anyone have any idea?
